workflow/rocoto_funcs/jedivar.py

```diff
@@ -47,11 +48,26 @@ def jedivar(xmlFile, expdir, do_spinup=False):
     ens_dep = ""
     if HYB_WGT_ENS != "0" and HYB_WGT_ENS != "0.0" and HYB_ENS_TYPE == "1":  # rrfsens
         RUN = 'rrfs'
+        ens_depm1=""
+        ens_depm2=""
+        ens_depm3=""
```

Comment:
Sorry for the picky comment, but I suggest we use `ens_dep1`, `ens_dep2`, `ens_dep3` directly without the letter `m`